.release-it.json

```diff
@@ -0,0 +1,32 @@
+{
+	"dry-run": true,
```

Comment:
dry-run is supposed to be overriden manually?

Reply:
Great catch - this was indeed an oversight and used for testing.